src/core/etl/tests/Flow/ETL/Tests/Unit/Row/Schema/Constraint/NotEmptyTest.php

```diff
@@ -29,7 +27,6 @@ public function test_not_empty_is_satisfied() : void
 
         $this->assertTrue($constraint->isSatisfiedBy(Entry::array('e', [1])));
         $this->assertTrue($constraint->isSatisfiedBy(Entry::boolean('e', false)));
-        $this->assertTrue($constraint->isSatisfiedBy(Entry::collection('e', new Entries())));
```

Comment:
this should be replaced with map first

Reply:
We don't have map yet, but the structure is already covered in the test (at end).